src/routes/view-content/guide-content/UwTranslationNotesContent.svelte

```diff
@@ -24,6 +26,7 @@
     let isLoading = false;
 
     $: fetchContent(guideResourceInfo);
+    $: isShowing && (audioPlayerKey = undefined);
```

Comment:
This fixes a bug where the Bible audio was displaying on the UW Translation Notes Guide tab.